<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spline_settings_widget_t</class>
 <widget class="QWidget" name="spline_settings_widget_t">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Line Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_interptype">
        <property name="text">
         <string>Interpolation Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="InterpolationDropDown"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_linethickness">
        <property name="text">
         <string>Line Thickness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="LineThicknessSpinner"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="joinTypes">
     <property name="title">
      <string>Join Type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="MiterJoin">
        <property name="text">
         <string>Miter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RoundJoin">
        <property name="text">
         <string>Round</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BevelJoin">
        <property name="text">
         <string>Bevel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_colors">
     <property name="text">
      <string>Colors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="ColorList"/>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
